### DISCUSSION TITLE
writing/2024/08/23/consulting-stack/

### DISCUSSION THREAD

**Discussion by giscus** [jxnl/blog#73] [General]
# writing/2024/08/23/consulting-stack/

Notes about my hobbies and other things I find interesting.

https://jxnl.co/writing/2024/08/23/consulting-stack/

**Comment by sidneyhori** [comment]
Great tips! I have worked for Accenture 26 yrs as a mgmt and tech consultant. I finally left 2yrs ago to have my own “boutique”. At Accenture I had all at my disposal, from EA to infinite marketing collateral. On my own I had nothing. I used some of the tools you recommended and will check out the others! A couple of other basic things I use: 

- company domain email
- landing page - I use wix
- linked.in
- insurance- some large corporations will ask you
- Azure - some companies will appreciate you using a “secure” cloud
- google workspace
- notion for collaboration / project mgmt
- have a DUNS number - some clients ask and it is free to have one
- ask clients to be able to use them as credentials
